certify-postgres-landregistry.properties

```diff
@@ -0,0 +1,139 @@
+# this source code form is subject to the terms of the mozilla public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## DataProvider plugin specific changes
+
+mosip.certify.issuer=CertifyIssuer
+mosip.certify.vcformat.vc.expiry=true
+# optional field to specify VC signature algo for CertifyIssuer, defaults to Ed25519Signature2020
+# other option: [RsaSignature2018]
+mosip.certify.issuer.vc-sign-algo=Ed25519Signature2020
+mosip.certify.issuer.pub.key=did:web:vharsh.github.io:DID:devcp2#key-0
+mosip.certify.issuer.uri=did:web:vharsh.github.io:DID:devcp2
+mosip.certify.issuer.svg.template.id=5b9c2a12-810a-7388-2dc8-13ee7ad88bac
+mosip.certify.integration.data-provider-plugin=PostgresDataProviderPlugin
+# Mimoto -- INJIMOB-2446
+mosip.certify.issuer.id.field.prefix.url=https://mosip.io/credential/
+
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+## Postgres DB specific config
+
+# maps OpenID scope to the DB query string where :id is fetched from the token's "sub" field
+mosip.certify.data-provider-plugin.postgres.scope-query-mapping={\
+    'land_statement_vc_ldp': 'select * from statement_data where statement_id=:id',\
+    'land_registry_vc_ldp': 'select * from registration_receipt_data where registration_id=:id'\
+  }
+
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
+mosip.certify.integration.scan-base-package=io.mosip.certify.postgresdataprovider.integration
+mosip.certify.integration.audit-plugin=LoggerAuditService
+## Path to the json file containing svg template
+mosip.certify.svg-templates=
+
+
+## softhsm pin
+mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.certify.cp2.security.pin}
+## ------------------------------------------- UseCase specific default overriding properties ------------------------------------------------------------
+mosip.certify.domain.url=https://${mosip.injicertify.cp2.host}
+mosip.certify.identifier=${mosip.certify.domain.url}
+mosip.certify.authorization.url=https://${mosip.esignet.mock.host}
+mosip.certify.database.name=inji_certify_cp2
+mosip.certify.key-values={\
+    'latest' : {\
```

Comment:
what about the backward version wellknowns ?

Reply:
It won't fit @vishwa-vyom as this VC's credentialSubject is almost ~10k characters long.

Reply:
We have to do something regarding this, otherwise it is inconsistent

Reply:
Can we add a else case to this if condition (https://github.com/mosip/inji-certify/blob/develop/certify-service/src/main/java/io/mosip/certify/services/VCIssuanceServiceImpl.java#L122) create a backward compatible response for vd11 and vd12 based on the config of latest ?

Reply:
It means if the version is vd11 or vd12 then it should go to else clause and there version should be "latest"?
```
if(issuerMetadata.containsKey(version))
            return issuerMetadata.get(version);
else {
          return issuerMetadata.get("latest");
 }
 ```

Reply:
No, I was proposing a backward compatibility solution to be built into the code. 
The idea is to get the latest from config and convert it into vd11 or vd12 version if vd11 and vd12 is not configured.
Basically we need to transform the latest vd11 and vd12 structure if the same is not configured.

I would also like to know more on the charter limitation part, I was not able to find details about the same online.

Reply:
This sounds like a new mini-feature Vishwa which will have to be tested by the QA team. Can we include this in the next release?

Reply:
ticket created: https://mosip.atlassian.net/browse/INJICERT-684

Reply:
For the above changes, 2 conversion methods for latest to vd11 and vd12 are added and PR is raised for the same:
Ref: https://github.com/mosip/inji-certify/pull/152